Header header
float64 x   # x coordinate (m)
float64 y   # y coordinate (m)
float64 psi # yaw angle (rad)
float64 v   # speed (m/s)
float64 lat # latitude (deg)
float64 lon # longitude (deg)
float64 a   # acceleration (m/s)
float64 df  # front steering angle (rad)
